test/units/testsuite-64.sh

```diff
@@ -140,6 +140,43 @@ EOF
     rm -fr "$mpoint"
 }
 
+testcase_simultaneous_events() {
+    local blockdev part partscript
+
+    blockdev="$(readlink -f /dev/disk/by-id/scsi-*_deadbeeftest)"
+    partscript="$(mktemp)"
+
+    if [[ ! -b "$blockdev" ]]; then
+        echo "ERROR: failed to find the test SCSI block device"
+        return 1
+    fi
+
+    cat >"$partscript" <<EOF
+$(printf 'name="test%d", size=2M\n' {1..50})
+EOF
+
+    # Initial partition table
+    sfdisk -q -X gpt "$blockdev" <"$partscript"
```

Comment:
Nitpick: use here string instead of a temp. file?

Reply:
The same file is used in the loop below, so I guess it's better to keep it in a file for readability.

Reply:
The partscript can be saved into a variable and the variable can be used in here string at both places...

Reply:
I see, I'll consider it with the next batch of patches.